src/components/Profile/index.tsx

```diff
@@ -0,0 +1,41 @@
+import styles from './Profile.module.css';
+
+function Profile() {
+    return (
+        <div className={styles['main-div']}>
+            <div className={styles['profile-data']}>
+                <div>
+                    <label>First name:</label>
+                    <p className={styles['border-bottom']}></p>
```

Comment:
Instead of `<p>` these should be `<input>` elements

Reply:
I set '<p>' because i didn't think in a way that we will be able in a future to edit our profile data. I will change it.